.github/workflows/reusable-publish-npm.yml

```diff
@@ -0,0 +1,31 @@
+name: Publish @bucketeer/js-client-sdk to npmjs
+
+on:
```

Comment:
Can you make it to be able to run this workflow manually as you did here?
https://github.com/bucketeer-io/android-client-sdk/blob/main/.github/workflows/publish-maven.yml#L3-L8

Reply:
Do you really have that usecase?

Reply:
In the main repository, we manually release binaries that are still in development for tests.
But at the moment, I don't think we will need it any time soon. So you can leave it as it is.

Reply:
When we manually release it for test purposes, we create a different branch and deploy it from that branch.
So, we won't need to change the package.json in the main branch.

Reply:
Should we just publish snapshot versions instead?

Reply:
I mean, nightly build or every time the main branch gets updated

Reply:
Yes, that would be great!

Reply:
I was thinking of publishing the snapshot only when it is needed.
But if there are no restrictions on publishing too many binaries to npm in a day, every time the main branch gets updated would be nice.

We publish it every time the main branch gets updated in the main repository.

Reply:
after searching for a bit, it turned out that:

- rate limit for publishing a package does not exist
- but it's difficult to obtain the next version number
  - at first, I thought release-please-action can provide this, but it [doesn't](https://github.com/google-github-actions/release-please-action/issues/684)


so I'm planning to create a separate workflow that you trigger manually

Reply:
added publish-snapshot workflow in [`0090251` (#42)](https://github.com/bucketeer-io/javascript-client-sdk/pull/42/commits/00902511661e030713a4d8064e58d90fc5d7db71)

Reply:
Nice work!

Reply:
Thanks!
Maybe later, we can look deeper at what release-please-action does and find out how they go through the commit log and decide the next version.